workflows/flows/analyse_study_tasks/raw_reads/run_rawreads_pipeline_via_samplesheet.py

```diff
@@ -101,7 +102,9 @@ def run_rawreads_pipeline_via_samplesheet(
             working_dir=rawreads_current_outdir,
             resubmit_policy=ResubmitIfFailedPolicy,
         )
+        close_old_connections()
     except ClusterJobFailedException:
+        close_old_connections()
         for analysis in rawreads_analyses:
             mark_analysis_as_failed(analysis)
```

Comment:
Hi @mberacochea noticed this is already merged, but had one question. 
Could this lead to a bottle neck if mark_analysis_as_failed is calling save() each time for each analysis record ?
Would it help to do a bulk update instead ?